frontend/src/pages/list/list.rs

```diff
@@ -89,10 +93,9 @@ pub fn update(
 }
 
 pub fn view(model: &Model, ctx: &Context) -> Node<Msg> {
-      
-
-    let filtered_lists = model.lists.iter()
-                                    .filter(|l| l.id.to_lowercase().contains(&model.filter))
+    let ctx_lists: Vec<crate::model::StoredList> = ctx.lists.values().cloned().collect();
+    let filtered_lists = ctx_lists.iter()
+                                    .filter(|l| Uuid::to_string(&l.id).to_lowercase().contains(&model.filter))
```

Comment:
What is this filter meant to achieve?
Since id's were strings, are the id's meant to be titles instead of id's?
If this is the case, I think it might be best to keep the UUIDs as identifiers of the lists in storage, don't think it's worthwhile using an arbitrary user string and checking its uniqueness all over the place.

Reply:
why not search in both?

```rust
.filter(|l| Uuid::to_string(&l.id).to_lowercase().contains(&model.filter) || l.description.to_lowercase().contains(&model.filter) )
```